package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "themes",
-	"version": "1.0.181",
+	"version": "1.0.182",
```

Comment:
Our deploy script handles versioning automatically for us, so this change doesn't need to be included in the PR.